### PR TITLE
[Feature] Calculate account balance at given timestamp

### PR DESCRIPTION
**Pull request by cahogan**
🍵 

Includes a rough, historical version of the assets vs. obligations graph.
image

### CODE REVIEW

File: code/svb-docker/python/svb_web/core/models.py [hunk 1]
```diff
@@ -65,6 +66,22 @@ class Account(models.Model):
...
     # Account balance at current timestamp can be calculated from previous anchor event
     # timestamp and anchor event account balance.
+    def get_balance(self, timestamp):
+        """
+        @brief Returns the balance of the account at the given timestamp. Backdates to the last anchor event.
+        And calculates the interest accrued since then.
+        @param[in] timestamp Datetime timestamp to calculate interest at.
+        @retval Account balance at the specified timestamp.
+        """
+        COMPOUNDING_INTERVAL_SECONDS = 1800
```
Comment (CoolNamesAllTaken) [inline code/svb-docker/python/svb_web/core/models.py:76]: Did we want to compound every second?

File: code/svb-docker/python/svb_web/core/models.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 from datetime import date
```
Comment (CoolNamesAllTaken) [inline code/svb-docker/python/svb_web/core/models.py:2]: Please convert from `datetime.item` format to `item.` format.

File: code/svb-docker/python/svb_web/core/templates/public/index.html [hunk 3]
```diff
@@ -1,44 +1,153 @@
...
+    <h2>We serve investors across houses, streets, and neighborhoods</h2>
+    <p>SVB is the go-to treat partner for discerning trick-or-treaters in the costume ecosystem and beyond.</p>
+    <div class="center-container">
+        <span class="dark-icon material-symbols-outlined">
+            house
+        </span>
+        <span class="dark-icon material-symbols-outlined">
+            house
+        </span>
+        <span class="dark-icon material-symbols-outlined">
```
Comment (CoolNamesAllTaken) [inline code/svb-docker/python/svb_web/core/templates/public/index.html:19]: love it

**Comment by cahogan** [comment]
@CoolNamesAllTaken this should be ready to merge, approve please

**Review by CoolNamesAllTaken — APPROVED** [review]
LGTM

**Pull request merged**